extensions/wikia/MyHome/MyHome.class.php

```diff
@@ -32,7 +32,7 @@ public static function storeInRecentChanges(RecentChange $rc, $data = array()) {
 
 		// If we have existing data packed into rc_params, make sure it is preserved.
 		if(isset($rc->mAttribs['rc_params'])){
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "rc_params" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)